endpoints.json

```diff
@@ -10,17 +10,18 @@
     }
   },
   "GET /api/articles": {
-    "description": "serves an array of all articles",
-    "queries": ["author", "topic", "sort_by", "order"],
+    "description": "serves an array of all articles ordered by date in descending order",
```

Comment:
I can see why you changed this description! You haven't misunderstood the task - this is left in as default because one of the later tasks will ask you to let this endpoint accept some queries. So nothing wrong really with removing it/you haven't misunderstood this task at all. Will make more sense when you see that later task 😄